cmd/loki/loki-docker-config.yaml

```diff
@@ -39,3 +39,14 @@ ruler:
 # If you would like to disable reporting, uncomment the following lines:
 #analytics:
 #  reporting_enabled: false
+
```

Comment:
Do we want compactors to always come with our docker config, now?

Reply:
Probably. I'll remove the compactor settings as they are set for testing and not for production.